File: Backend/lib/src/main/kotlin/com/github/bratek20/ostium/carddrawer/context/Impl.kt
package com.github.bratek20.ostium.carddrawer.context

import com.github.bratek20.architecture.context.api.ContextBuilder
import com.github.bratek20.architecture.context.api.ContextModule

import com.github.bratek20.ostium.carddrawer.api.*
import com.github.bratek20.ostium.carddrawer.impl.*

class CardDrawerImpl: ContextModule {
    override fun apply(builder: ContextBuilder) {
        builder
            .setImpl(CardDrawerApi::class.java, CardDrawerApiLogic::class.java)
    }
}

File: Frontend/Logic/B20/Frontend/Traits/Context/Impl.cs
using B20.Architecture.Contexts.Api;
using B20.Frontend.Element;
using B20.Frontend.Traits.Impl;

namespace B20.Frontend.Traits.Context
{
    public class TraitsImpl: ContextModule
    {
        public void Apply(ContextBuilder builder)
        {
            builder
                .SetImpl<TraitFactory, TraitFactoryLogic>()
                .SetClass<WithRect>()
                .SetClass<Clickable>();
        }
    }
}